playbooks/hc-ansible-deployment/node_replace_inventory.yml

```diff
@@ -0,0 +1,10 @@
+cluster_node:
+  hosts:
+   # mention whatever host is provided in gluster_maintenance_cluster_node or in gluster_maintenance_cluster_node_2 either one.
+   <host2-backend-network-FQDN>:
+
+  vars:
+    gluster_maintenance_old_node: <host1-backend-network-FQDN>
+    gluster_maintenance_new_node: <host1-backend-network-FQDN>
+    gluster_maintenance_cluster_node: <host2-backend-network-FQDN>
+    gluster_maintenance_cluster_node_2: <host3-backend-network-FQDN>
```

Comment:
Better names should be good:
    gluster_maintenance_old_node: <host-backend-network-FQDN>
    gluster_maintenance_new_node: <host-backend-network-FQDN>
    gluster_maintenance_cluster_node: <active-host1-backend-network-FQDN>
    gluster_maintenance_cluster_node_2: <active-host2-backend-network-FQDN>